src/Honeybee_ExtrudeWindows.py

```diff
@@ -0,0 +1,97 @@
+#
+# Honeybee: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
+# 
+# This file is part of Honeybee.
+# 
+# Copyright (c) 2013-2015, Sarith Subramaniam <sarith@sarith.in> 
+# Honeybee is free software; you can redistribute it and/or modify 
+# it under the terms of the GNU General Public License as published 
+# by the Free Software Foundation; either version 3 of the License, 
+# or (at your option) any later version. 
+# 
+# Honeybee is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of 
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+
+
+"""
+Extrude pseudo walls from window polygons. This component has only been tested with rectangular windows.
+
+-
+Provided by Honeybee 0.0.58
+    
+
+    Args:
+        _glazings: Any number of glazing polygons.
+        _thickness: Thickness of the window. Can be a single number or a list of numbers. If its a list then the list should be equal to the number of glazings. Default value is 1.0.
+    Returns:
+        GlazingWalls: Geometric representations of glazing walls.
+"""
+
+
+from __future__ import print_function
+
+ghenv.Component.Name = "Honeybee_Extrude_Windows"
```

Comment:
I know we are not consistent on this but since the name will show up when you search in Grasshopper underline is usually only used after Honeybee and there will be no _ between the rest. In your case it will be Honeybee_Extrude Windows. Nilckname on the other hand is camelCase.
```python
ghenv.Component.Name = "Honeybee_Run Daylight Simulation"
ghenv.Component.NickName = 'runDaylightAnalysis'
```
@chriswmackey do we agree on this? should we add this to graphic standards?